py-polars/polars/dataframe/frame.py

```diff
@@ -3390,8 +3390,8 @@ def write_parquet(
         compression: ParquetCompression = "zstd",
         compression_level: int | None = None,
         statistics: bool = False,
-        row_group_size: int | None = None,
-        data_page_size: int | None = None,
+        row_group_size: int | None = 512 ** 2,
```

Comment:
The behavior is currently different for `polars` and `pyarrow`: `pyarrow` uses a single row group with `row_group_size is None` while `polars` creates `512 ** 2` row groups. I would argue that the `pyarrow` interface is superior as it allows, for example, `lf.collect().write_parquet("file.parquet", row_group_size=None)`.